src/user.js

```diff
@@ -11,40 +14,76 @@ const getUserGeo = () => {
           lat: result.coords.latitude,
           long: result.coords.longitude
         }
-
-        localStorage.setItem('userData', JSON.stringify(data))
         return resolve(data)
       })
     }
   })
 }
 
-// Set flag if user has joined to app
+/**
+ * Set flag if user has joined to app
+ */
 const setUserHasJoined = () => {
   localStorage.setItem('userHasJoined', JSON.stringify(true))
 }
 
-// Check if user has joined to app
+/**
+ * Check if user has joined to app
+ */
 const checkHasJoined = () => {
   return !!localStorage.getItem('userHasJoined')
 }
 
-// Check geo
-const checkUserData = () => {
-  return !!localStorage.getItem('userData')
+/**
+ * Set user geo data
+ * @param {object} data
+ */
+const setUserGeoDataToStorage = data => {
+  localStorage.setItem('userGeoData', JSON.stringify(data))
+}
+
+/**
+ * Get user geo data from localstorage
+ */
+const getUserGeoDataFromStorage = () => {
+  return JSON.parse(localStorage.getItem('userGeoData'))
+}
+
+/**
+ * Check if user has geo data
+ */
+const checkUserGeoData = () => {
+  return !!localStorage.getItem('userGeoData')
+}
+
+/**
+ * Fetch user geo data
+ * @return {object} user geo data
+ */
+const fetchUserGeoData = async () => {
+  const data = await getUserGeoDataFromDevice()
+  setUserGeoDataToStorage(data)
+  return data
 }
 
-// Get user data
-const getUserData = async () => {
-  return checkUserData()
-    ? JSON.parse(localStorage.getItem('userData'))
-    : getUserGeo()
+/**
+ * Get user data from storage or user device
+ * @param {boolean} force
+ * @return {object} user geo data
```

Comment:
@raulghm esto debe ser:

```suggestion
 * @return {Promise<object>} user geo data
```